### PR TITLE
Update how-to-write-your-first-technical-book.md

### LINKED ISSUE #1137: [Auto]（如何写自己的第一本技术书籍）
- 原文网址：[原文标题](https://www.freecodecamp.org/news/how-to-write-your-first-technical-book/)
- MarkDown 文件：https://github.com/freeCodeCamp/news-translation/edit/main/chinese/articles/how-to-write-your-first-technical-book.md

### PR DESCRIPTION
**Pull request by Narcissus91**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #1137

### CODE REVIEW

**Comment by Narcissus91** [comment]
@miyaliu666  这篇翻译好了

File: chinese/articles/how-to-write-your-first-technical-book.md [hunk 1]
```diff
@@ -3,272 +3,273 @@
...
 ```
 
-### About Me Page
+### 关于我的页面
 
-In the **About Me** section of my book, I divided the content into two columns: a brief about me and a profile picture.
+在我的书中 **关于我**部分，我把内容分为了两栏：一栏是关于我的简介，另一栏是一张个人照片。
 
-It took me a while to realize the full capabilities of the markdown format. We can add plain `html` code in it. Here's what my "about me" page says:
+我花了一段时间才了解Markdown格式的全部功能。我们可以在Markdown里添加普通的`html`代码。下面是我的 "关于我 "页面的内容：
```
Comment (miyaliu666) [inline chinese/articles/how-to-write-your-first-technical-book.md:272]: plain html 可以理解为原生 html

File: chinese/articles/how-to-write-your-first-technical-book.md [hunk 2]
```diff
@@ -313,142 +314,141 @@ wrapping a complex Hyperledger Fabric into the easy to use Hyperledger Composer.
...
+## 许可协议
 
-I wrote this book out of curiosity and for fun. So, I wanted it to be free, and open-source, but I didn't want others to monetize it. Without a license, there are no restrictions.
+我写这本书是出于好奇心和乐趣。所以，我希望它是免费的，而且是开源的，但我不希望有人利用这本书来谋利。没有许可协议，就没有约束。
 
-I searched for a while and found a great answer on StackOverflow regarding free licenses, [Creative Commons Licenses](https://creativecommons.org/licenses/).
+我搜索了一段时间，在StackOverflow上发现了一个关于免费许可协议的很好回答，即[知识共享版权许可协议（Creative Commons Licenses）](https://creativecommons.org/licenses/)。
 
-> **Creative Commons is a nonprofit organization that helps overcome legal obstacles to the sharing of knowledge and creativity to address the world’s pressing challenges.**
+> **知识共享基金会（Creative Commons）是一个非营利组织，帮助解决知识分享和创造力有关的法律困境，应对世界上（译者注：版权方面的）紧迫的挑战。**
```
Comment (miyaliu666) [inline chinese/articles/how-to-write-your-first-technical-book.md:362]: pressing 作名词有出版社的意思。我理解这里作者误把它当动词的出版使用了。

**Comment by miyaliu666** [comment]
翻译得非常好，我写了一点点建议，谢谢 @Narcissus91 ！

**Pull request merged**